src/ReleaseHistory.md

```diff
@@ -13,7 +13,8 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
-## **v4.0.0-rc5** UNRELEASED
+## **v4.0.0** UNRELEASED
+* BUG: Fix `PdbLoad` not working for `--trace`. [821](https://github.com/microsoft/binskim/pull/821)
```

Comment:
>BUG [](http://example.com/codeflow?start=2&length=3)

Remember in the release notes to put yourself in the perspective of the customer. What evidence of this problem would they have seen? They will look for that evidence here. 

For exceptions, the typical thing to search on is the type of the exception and the top of the stack where it occurs. Notice how much more useful information is encoded in the line below. Please do think about this particular code review note, it is a comment that I have to frequently make. 

Great fix! And a nice job with this change overall, thanks very much

Fix unhandled `ArgumentException` in `Enum.TryParse` on passing `PdbLoad` value to `--trace` command-line argument.

Reply:
yes, fixed.